pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -986,3 +1037,62 @@ func (b *SlirpPodInterface) setCachedVIF(pid, name string) error {
 func (s *SlirpPodInterface) setCachedInterface(pid, name string) error {
 	return nil
 }
+
+func (s *SlirpPodInterface) createTapDevice() (string, error) {
+	return "", nil
+}
+
+func (s *SlirpPodInterface) configureTapDevice() error {
+	return nil
+}
+
+func createTapDevice() (string, error) {
+	tapName := generateTapDeviceName()
+	args := []string{"tuntap", "add", "mode", "tap", "user", "qemu", "group", "qemu", "name", tapName}
+	cmd := exec.Command("ip", args...)
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		log.Log.Reason(err).Criticalf("Failed to create tap device %s. Reason: %s", tapName, out)
+		return "", err
+	}
+	log.Log.Infof("Created tap device: %s", tapName)
```

Comment:
What's the device that's being generated here?

Reply:
Hm, sorry but I don't understand the question. You mean the type of device ?... The name ?...